docs/apache-airflow/concepts/tasks.rst

```diff
@@ -163,6 +163,10 @@ If you want to disable SLA checking entirely, you can set ``check_slas = False``
 
 To read more about configuring the emails, see :doc:`/howto/email-config`.
 
+.. note::
+
+    Only scheduled tasks will be checked against SLA. For example, manually triggered tasks will not invoke an SLA miss. For more information on ``schedule_interval`` values see :doc:`DAG Run </dag-run>`.
+
```

Comment:
Checking this again, I think we need a note about this in `UPDATING.md` - can you do that please @john-jac ?

cc @jedcunningham (Release manager for 2.2.2)

Reply:
Also, I am not sure if we should actually restrict SLAs just for scheduled DAG Runs. We can change SLA to work on "timedelta" from the start date.

Thoughts @dstandish @jedcunningham @ashb @uranusjr ?

Reply:
> Checking this again, I think we need a note about this in `UPDATING.md` - can you do that please @john-jac ?

Sure!  https://github.com/apache/airflow/pull/19476

Reply:
I'm not sure this needs to be in `UPDATING.md`. Unless I'm mistaken, this just doesn't blow up and instead logs, whereas 2.1.4 is simply silent about it? This isn't a change in behavior.

Reply:
Aah right.

I think this had stopped working a long time ago:

https://github.com/apache/airflow/blob/1.10.15/airflow/jobs/scheduler_job.py#L462-L468